ext/vscode/src/commands/init.ts

```diff
@@ -54,8 +54,10 @@ export async function init(context: IActionContext, selectedFile?: vscode.Uri, a
         .withArg('init');
 
     if (useExistingSource) {
+        context.telemetry.properties.useExistingSource = 'true';
```

Comment:
#3845 removed this data point and I forgot to add it back, so adding it back here.